Galaxy.html

```diff
@@ -0,0 +1,68 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<title>Galaxy</title>
+		<link rel="stylesheet" href="visual.css">
+		<link rel="shortcut icon" href="icon.ico" type="image/x-icon">
+	</head>
+	<body>
+	<header class="header">
+		
+		<nav class="bar">
+			<a href="#">Home</a>
+			<a href="#">About us</a>
+			<a href="#">Charity</a>
+			<a href="#">Contacts</a>
+			<p>UNKNOWN UNIVERSE</p>
+		</nav>
+		
+	</header>
+		
+		<div class="whole-universe">
+			<aside class="voating">
+					<section><input type="radio" name="planet" value="1"><i>Uranus</i></section>
```

Comment:
**section** here is redundant, you can leave **ul + li** inside wrapper aside or maybe just leave **inputs**

Reply:
> **section** here is redundant, you can leave **ul + li** inside wrapper aside or maybe just leave **inputs**

change, leave just inputs